specs/server/exports.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable prefer-arrow-callback, func-names, import/named */
+/* eslint-env mocha */
+import { expect } from 'chai';
+import {
+    DDP_APOLLO_SCHEMA_REQUIRED,
+    setup,
+    createGraphQLPublication,
+    setupHttpEndpoint
+} from '../../server';
+
+describe('Server exports', function () {
+  it('exports the DDP_APOLLO_SCHEMA_REQUIRED constant', function () {
+    expect(DDP_APOLLO_SCHEMA_REQUIRED).to.be.a('string');
+  });
+
+  it('exports the setup function', function () {
+    expect(setup).to.be.a('function');
+  });
+
+  it('exports the createGraphQLPublication function', function () {
```

Comment:
Figured while I was here, I'd add some simple tests for all of the exports done on the server.